config/bradesco_config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import "os"
+
+type BradescoConfig struct {
+	URLBradescoShopFacil  string
+	URLBradescoNetEmpresa string
+}
+
+func getBradescoConfig() BradescoConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀

```suggestion
func getBradescoConfig() *BradescoConfig {
```